guides/upstream_filename_to_satellite_link.json

```diff
@@ -0,0 +1,19 @@
+{
+  "build/Administering_Project/index-katello.html": "administering_red_hat_satellite",
+  "build/Configuring_Load_Balancer/index-katello.html": "configuring_capsules_with_a_load_balancer",
+  "build/Deploying_Project_on_AWS/index-katello.html": "deploying_red_hat_satellite_on_amazon_web_services",
+  "build/Installing_Proxy/index-katello.html": "installing_capsule_server",
+  "build/Installing_Server/index-katello.html": "installing_satellite_server_in_a_connected_network_environment",
+  "build/Installing_Server_Disconnected/index-katello.html": "installing_satellite_server_in_a_disconnected_network_environment",
+  "build/Managing_Configurations_Ansible/index-katello.html": "managing_configurations_using_ansible_integration_in_red_hat_satellite",
+  "build/Managing_Configurations_Puppet/index-katello.html": "managing_configurations_using_puppet_integration_in_red_hat_satellite",
+  "build/Managing_Content/index-katello.html": "managing_content",
+  "build/Managing_Hosts/index-katello.html": "managing_hosts",
+  "build/Provisioning_Hosts/index-katello.html": "provisioning_hosts",
+  "build/Planning_for_Project/index-katello.html": "overview_concepts_and_deployment_considerations",
+  "build/Tuning_Performance/index-katello.html": "tuning_performance_of_red_hat_satellite",
+  "build/Managing_Security_Compliance/index-katello.html": "managing_security_compliance",
+  "build/Upgrading_Project/index-katello.html": "upgrading_connected_red_hat_satellite_to_6.15",
+  "build/Upgrading_Project_Disconnected/index-katello.html": "upgrading_disconnected_red_hat_satellite_to_6.15",
```

Comment:
If this includes the version in the filename then we'll need to make sure that this is updated during Satellite version bumping.

Reply:
Yes, that would make sense. It will be detected on the first run of the tests against the new TOC, since the links to the next version will be missing.